manifests/dev/vsphere-7.0u2/vanilla/rbac/vsphere-csi-node-rbac.yaml

```diff
@@ -0,0 +1,29 @@
+kind: ServiceAccount
```

Comment:
Is there a CSI upgrade impact due to this? Do we need a doc that talks about how to manually upgrade CSI until we have CSI operator?

Reply:
IMO CSI upgrade involves re-creating the deployment YAMLs again, so this should not be a problem. During my testing, I used an existing environment and applied the new YAMLs and I did not face any problems

Reply:
Did you test upgrading PVCSI? Does that delete the old objects and create new ones?

Reply:
You mean I should follow the upgrade process that TKG uses to upgrade its clusters? I would have to do it manually. I just replaced the YAMLs in an existing environment and checked

Reply:
Can you confirm if TKG cluster upgrade deletes the old yaml and creates the new one?

Reply:
> When the distribution version is changed on a TanzuKubernetesCluster, the add-ons controller will pull the new object manifests and create or update those objects in the workload cluster

This confirms that TKG add-ons controller creates/updates the manifests of CSI during an upgrade. Refer to https://gitlab.eng.vmware.com/guest-clusters/tkg-service-tools/blob/main/docs/cluster-lifecycle/rolling-update.md#add-ons-update for more details